pkg/arvo/sys/hoon.hoon

```diff
@@ -7509,7 +7489,7 @@
 ::
 ++  loot
   ~/  %loot
-  |=  [cog=term dom=(map term tome)]
+  |=  [cog=term dom=(map term (pair ~ tome))]  ::NOTE  ~ for jet-compat
```

Comment:
Since we're bumping kelvins across the board and get a new jet dashboard anyway, we can change the signature here and update the jet to match.